src/models/TextCompressor.cpp

```diff
@@ -0,0 +1,47 @@
+#include "../doctest.h"
+#include "TextCompressor.h"
+
+TextCompressor::TextCompressor(int id){
+  this->id = id;
+}
+
+int TextCompressor::fitString(std::string text, long timestamp){
+  if(this->timestamps.empty()){
+    this->string = text;
+    this->timestamps.push_back(timestamp);
```

Comment:
Lav om til at returner bool, i stedet for c metoden med int

Reply:
Fixed.